### PR TITLE
CHANGELOG relesae notes for 1.5.5 & 1.5.6

### LINKED ISSUE #2569: CHANGELOG Release Notes for 1.5.5
* Malformed rewards link under "chrome://about" #2534
* several NTP SI fixes and changes #2533

### LINKED ISSUE #2570: CHANGELOG Release Notes for 1.5.6
* notification for private tabs mentions 'incognito' #2559
* context menu on image/link says 'Incognito' instead of private #2558
* Upgrade to Chromium 80.0.3987.99 #2522
* Claim multiple grants from the panel. #2509
* Block notifications bridged to Wear OS.Issue #7759 #2493

### PR DESCRIPTION
**Pull request by kjozwiak**
Fix #2570 Fix #2569

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [1.5.6](https://github.com/brave/browser-android-tabs/releases/tag/v1.5.6)
+
+ - Disabled bridging mode for notifications on Wear OS devices. ([#2493](https://github.com/brave/browser-android-tabs/pull/2493))
+ - Fixed not being able to claim multiple ad grants using the rewards panel. ([#2509](https://github.com/brave/browser-android-tabs/pull/2509))
+ - Fixed context menus using "Incognito" instead of "Private". ([#2558](https://github.com/brave/browser-android-tabs/issues/2558))
```
Comment (srirambv) [inline CHANGELOG.md]: Probably better to say replaced instead of fixed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+ - Fixed private tabs notification using "Incognito" instead of "Private". ([#2559](https://github.com/brave/browser-android-tabs/issues/2559))
```
Comment (srirambv) [inline CHANGELOG.md]: same as above

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+ - Upgraded to Chromium 80.0.3987.99. ([#2522](https://github.com/brave/browser-android-tabs/issues/2522))
+
+## [1.5.5](https://github.com/brave/browser-android-tabs/releases/tag/v1.5.5)
+
+ - Improved several visual aspects of New Tab Page Sponsored Images. ([#2533](https://github.com/brave/browser-android-tabs/issues/2533)) 
+ - Disabled New Tab Page Sponsored Images for devices running Android 4 or lower. ([#2533](https://github.com/brave/browser-android-tabs/issues/2533))
```
Comment (srirambv) [inline CHANGELOG.md]: Not specific to sponsored images so better to say "New Tab Page Images"

Comment (kjozwiak) [reply CHANGELOG.md]: I think leaving it as sponsored is fine as most of the fixes were related to NTP SI. Example:

```
* Add bottom gradient on NTP for better visibility of the sponsored logo
* Update size of the sponsored logo
* Update centerpoint for NTP images calculation
```

I guess the only NTP change was the centerpoint but that issue was mostly for NTP SI.

File: CHANGELOG.md (hunk 1 above)
Comment (kjozwiak) [reply CHANGELOG.md]: Changed it to `Replace`

File: CHANGELOG.md (hunk 2 above)
Comment (kjozwiak) [reply CHANGELOG.md]: Changed to `Replace`

File: CHANGELOG.md (hunk 3 above)
Comment (kjozwiak) [reply CHANGELOG.md]: BTW, if you feel strongly about keeping it NTP, we can change but seems like that issue was basically for NTP SI. Either way works TBH.

Comment (srirambv) [reply CHANGELOG.md]: Disabling NTP SI sounds like we only disabled SI on Android 4 but we totally removed images all together from Android 4. 

Issue mentions 
> Remove normal background images and sponsored images for devices lower than android 5.

Also this clearly says no images setting so not specific to NTP SI
> Remove settings option for New Tab Page for devices lower than android 5.

**Pull request merged**